src/OrchardCore.Modules/OrchardCore.Contents/Views/Admin/Edit.cshtml

```diff
@@ -8,11 +8,13 @@
     ContentItem contentItem = Model.ContentItem;
     var contentTypeDefinition = await ContentDefinitionManager.GetTypeDefinitionAsync(contentItem.ContentType);
     var typeDisplayName = contentTypeDefinition?.DisplayName ?? contentItem.ContentType.CamelFriendly();
+
+    var returnUrl = Context.Request.Query["returnUrl"];
```

Comment:
Please revert this change coz there's one occurrence in this file

Reply:
done